dist/main.js

```diff
@@ -2,27 +2,31 @@
 const core = require('@actions/core');
 const twilio = require('twilio');
 async function run() {
-    const fromPhoneNumber = core.getInput('fromPhoneNumber');
-    const toPhoneNumber = core.getInput('toPhoneNumber');
+    const from = core.getInput('fromPhoneNumber');
```

Comment:
Not sure why this is showing in diff?
It was merged in https://github.com/twilio-labs/actions-sms/pull/10